templates/Web/_composition/Flask/Page.Flask.AddEndPoints/.template.config/template.json

```diff
@@ -10,6 +10,7 @@
     "wts.type": "composition",
     "wts.platform": "Web",
     "wts.version": "1.0.0",
+    "wts.compositionOrder": "0",
```

Comment:
just out of curiosity, where is `compositionOrder` used?

Reply:
It's used in core in the following areas:
https://github.com/microsoft/CoreTemplateStudio/blob/e97c1c579b2c0719b1abaaf30e2023a3a0089d16/code/src/CoreTemplateStudio/CoreTemplateStudio.Core/Templates/ITemplateInfoExtensions.cs#L298-L311

https://github.com/microsoft/CoreTemplateStudio/blob/e97c1c579b2c0719b1abaaf30e2023a3a0089d16/code/src/CoreTemplateStudio/CoreTemplateStudio.Core/Gen/GenComposer.cs#L148-L191